.github/PULL_REQUEST_TEMPLATE/control_plane_template.md

```diff
@@ -4,7 +4,7 @@
 
 Please understand this diagram before proceeding. It explains how to get your PR approved & merged.
 
-![diagram](https://github.com/Azure/azure-rest-api-specs/assets/4429827/029a11fd-eaf9-4865-9d77-a16b5ea5eb26)
+![diagram](https://github.com/Azure/azure-rest-api-specs/assets/20296335/93ad7897-061a-4724-8a51-dcb797d5a6e0)
```

Comment:
Hey I see that in the new diagram there are red squiggles / underlines under some label names. I think it makes the diagram a little bit harder to read and would be nice to remove. But if you don't want to do it now, let's just remember to take care of this on next diagram update.

Reply:
Removed.